### ISSUE TITLE
Error with newer versions of Grafana

### ISSUE THREAD

**Issue by negon** [pueteam/datasource-plugin-solr#12]
With newer versions of Grafana I can't test the connection with http://localhost:8983, I need to add /solr to pass the test. But then in Explorer /solr is added again. 
If I remove /solr in datasource settings, it gets the SOLR fields in Explorer, but there is always an error saying:
"Cannot read properties of undefined (reading 'meta')"

**Comment by ishtartec** [comment]
What version of Grafana are you using?

**Comment by negon** [comment]
I have tried with 7 and 8

**Comment by negon** [comment]
@ishtartec what versions does the plugin work with? 
The plugin that's available with the Grafana CLI, id depending on Grafana version 3.x.x. But that version is not available for download anymore.

**Comment by ishtartec** [comment]
It should work with newer versions. I have merged a pull request. Can you download the latest version and retry?. And also, if you ignore that error, is it allowing you to save and continue?

**Comment by negon** [comment]
I have pulled the code and tried again, but it does not seem to help.
I can ignore the error and find the SOLR datasource in Explorer but the "Cannot read properties of undefined (reading 'meta')" is still there.

Here is a screenshot of the error, the fields are found automatically, so the SOLR connection is there:
![Screenshot from 2022-05-18 09-29-05](https://user-images.githubusercontent.com/4978938/168982664-dcc8f9ed-8af4-4842-b82e-76e6dd34295e.png)

Thanks for looking into it, it really means a lot!

**Comment by ishtartec** [comment]
Can you show the output when clicking the inspector button?
If you can also provide the solr version and if it is cloud mode or single solr instance?

**Comment by negon** [comment]
The solr version is 6.3.0
Its running standalone, but for some reason the Query history is:
{"target":"*:*","type":"timeserie","time":"date_tdt","fields":"sensor_s","groupEnabled":"standard","groupValueField":"","groupByField":"","groupLimit":100000,"collection":"clickcounter","solrCloudMode":true}
I have not checked Solrcloud in settings.

I am attaching the output of Query inspector and JSON 
[json.txt](https://github.com/pueteam/datasource-plugin-solr/files/8714402/json.txt)
[query.txt](https://github.com/pueteam/datasource-plugin-solr/files/8714403/query.txt)

**Comment by ishtartec** [comment]
The output seems ok. I'm just refactoring a bit to adapt the datasource to the latest versions of grafana. Give me a couple of days to fix things.

**Comment by negon** [comment]
hi @ishtartec Have you had time to do the refactoring? We would really mean a lot if we could use the plugin.

**Comment by ishtartec** [comment]
Yes, I have almost finished. It's more complex than expected, as the grafana team has migrated from Angular to react components. I had to rewrite most of the code. I guess I will finish during the next week. I can publish a beta release in the mean time so you can try before, if you are interested.

**Comment by negon** [comment]
That sounds great, didn't know they had migrated from Angular.
I would be very interested to test the beta version!

**Comment by ishtartec** [comment]
I have created a new branch: devel
It contains all the rework done until now. It has some limitations. Group queries are not implemented yet, and fields list accept a single numerical value, instead of multiple values. But it should work. Remember to set the collection name in the DataSource configuration.

Let me know if you need help for deploying the plugin.

![image](https://user-images.githubusercontent.com/11651886/171590942-fa84dd17-5f23-4778-9a87-3f347523f35e.png)

**Comment by negon** [comment]
It definitely is working much better now, my data doesn't fit the single numerical needed right now, but looking very promising.

**Comment by OkkeKlein** [comment]
Im trying to get this running, but It complains about plugin not being signed. What are the steps to deploy?

**Comment by kashisT** [comment]
same thing I am facing as well when I did git clone I get to see that the grafana is disabling the plugin by saying that the signature is not verified.

**Comment by 11iamsid** [comment]
Even i am facing the same the issue after cloning  the repo from branch devl. Anybody was able to find a solution .

**Comment by Ella-eln** [comment]
We experienced this issue too. Is there a plan to merge these changes into main soon?
What is correctly missing?
